acceptance/tests/load_libfacter.rb

```diff
@@ -1,31 +1,44 @@
-test_name 'Ruby can load libfacter'
+test_name 'C100161: Ruby can load libfacter' do
```

Comment:
When testing the `require 'facter'` on Windows it works, but due to a different `facter` lib file being in the $LOAD_PATH.
```
Administrator@cl29d7nvck9bawi ~
$ env PATH="/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/sys/ruby/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/sys/ruby/bin:${PATH}" ruby -e 'puts $LOAD_PATH'
C:/Program Files/Puppet Labs/Puppet/sys/ruby/lib/ruby/site_ruby/2.1.0
C:/Program Files/Puppet Labs/Puppet/sys/ruby/lib/ruby/site_ruby/2.1.0/x64-msvcrt
C:/Program Files/Puppet Labs/Puppet/sys/ruby/lib/ruby/site_ruby
C:/Program Files/Puppet Labs/Puppet/sys/ruby/lib/ruby/vendor_ruby/2.1.0
C:/Program Files/Puppet Labs/Puppet/sys/ruby/lib/ruby/vendor_ruby/2.1.0/x64-msvcrt
C:/Program Files/Puppet Labs/Puppet/sys/ruby/lib/ruby/vendor_ruby
C:/Program Files/Puppet Labs/Puppet/sys/ruby/lib/ruby/2.1.0
C:/Program Files/Puppet Labs/Puppet/sys/ruby/lib/ruby/2.1.0/x64-mingw32

Administrator@cl29d7nvck9bawi ~
$ find "/cygdrive/c/Program Files/Puppet Labs/Puppet/sys/ruby/lib/ruby" -name facter.rb -print
/cygdrive/c/Program Files/Puppet Labs/Puppet/sys/ruby/lib/ruby/vendor_ruby/puppet/indirector/facts/facter.rb

Administrator@cl29d7nvck9bawi ~
$ env PATH="/cygdrive/c/Program Files (x86)/Puppet Labs/Puppet/sys/ruby/bin:/cygdrive/c/Program Files/Puppet Labs/Puppet/sys/ruby/bin:${PATH}" ruby -e 'require "facter"'
C:/Program Files/Puppet Labs/Puppet/sys/ruby/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require': cannot load such file -- facter (LoadError)
        from C:/Program Files/Puppet Labs/Puppet/sys/ruby/lib/ruby/2.1.0/rubygems/core_ext/kernel_require.rb:54:in `require'
        from -e:1:in `<main>'

```